index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+
+
+
+The states sent by the receiver include the following individual state values,
+each of which is optional.  This allows the receiver to update the controller
+about one state value or many state values at once without having to specify all
+state values every time.  A non-present state value indicates the state has not
+changed.
+
+: supports
+:: The controls the receiver supports.  These may differ for different [=media
+    resource=]s and should not changes unless the [=media resource=] changes.  If
+    not present in the intial state in the remote-playback-start-response
+    message, support for nothing is assumed.
+
+: source
+:: The current [=media resource=] URL. See
+    [HtmlMediaElement.currentSrc](https://html.spec.whatwg.org/multipage/media.html#dom-media-currentsrc).
+    Must be present in the intial state in the remote-playback-start-response
+    message.
+
+: loading
+:: The state of network activity for loading the [=media resource=].  See
+    [HtmlMediaElement.networkState](https://html.spec.whatwg.org/multipage/media.html#dom-media-networkstate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be empty.
+
+: loaded
+:: The state of the loaded media (whether enough is loaded to play).  See
+    [HtmlMediaElement.readyState](https://html.spec.whatwg.org/multipage/media.html#dom-media-readystate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be nothing.
+
+: error
+:: A major error occurred which prevents the remote playback from continuing. See
+    [HtmlMediaElement.error](https://html.spec.whatwg.org/multipage/media.html#dom-media-error) and 
+    [HtmlMediaElement media error codes](https://html.spec.whatwg.org/multipage/media.html#concept-mediaerror-code).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed there is no such major error.
+
+: epoch
+:: The "zero time" of the media timeline.  See
+    [HtmlMediaElement's timeline offset](https://html.spec.whatwg.org/multipage/media.html#timeline-offset) and 
+    [HtmlMediaElement.getStartDate()](https://html.spec.whatwg.org/multipage/media.html#dom-media-getstartdate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be unknown.
+
+: duration
+:: The duration of the media timeline.  See
+    [HtmlMediaElement.duration](https://html.spec.whatwg.org/multipage/media.html#dom-media-duration).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be unknown.  For unbounded streams, it may be "forever".
+
+: buffered-time-ranges
+:: The time ranges for which media has been buffered.  See
+    [HtmlMediaElement.buffered](https://html.spec.whatwg.org/multipage/media.html#dom-media-buffered).
+
+: played-time-ranges
+:: The time ranges for which media has been played.   See
```

Comment:
I feel like the whatwg description is clearer, it's not obvious what "been played" means to me here. Plus, I generally dislike variable descriptions that are just the variable name with linking verbs.

Maybe something like:
> :: The interval of time that content playback has already reached

Reply:
Good point.  I did something in between you text and the WHATWG text.